pkg/resource/agents-toggle/reconciler.go

```diff
@@ -44,6 +44,11 @@ func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Inter
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
+	// If the observability-bundle version is too old, we don't need to do anything.
```

Comment:
What we do here is too disabled both grafana agent and alloy-event apps, not not do anything because we still need to toggle promtail right ?

Reply:
Oh ok my bad. Since 0.9.0 is such an old version of the observability-bundle I wasn't even sure that promtail was already managed by the logging-operator.

Reply:
I updated the handling of this specific case. WDYT about it ?